domain/dagconfig/params.go

```diff
@@ -287,7 +287,7 @@ var TestnetParams = Params{
 	RPCPort:     "42210",
 	DefaultPort: "42211",
 	DNSSeeds: []string{
-		"testnet-1.karlsencoin.com",
+		"testnet-1-dnsseed.karlsencoin.com",
```

Comment:
@okilisan They are not yet running, is it ok?

Reply:
I'm setting it up. It should be ok